docs/management/advanced-options.asciidoc

```diff
@@ -22,6 +22,7 @@ compatible with other configuration settings. Deleting a custom setting removes
 .Kibana Settings Reference
 [horizontal]
 `query:queryString:options`:: Options for the Lucene query string parser.
+`query:allowLeadingWildcards`:: When set, * is allowed as the first character in a query clause. Currently only applies when experimental query features are enabled in the query bar. To disallow leading wildcards in basic lucene queries, use query:queryString:options.
```

Comment:
If the convention here that `query:queryString` will be for lucene settings, while `query:<anything else>` will be for Kuery?

Reply:
Yeah I suppose so, hopefully we won't be adding any more settings specifically for lucene and eventually we'll remove `query:queryString:options`